src/week-view.h

```diff
@@ -28,7 +28,9 @@ void week_view_remove_event(WeekView* wv, Event* vevent);
 void week_view_focus_event(WeekView* wv, Event* event);
 void week_view_add_calendar(WeekView* widget, Calendar* cal);
 void week_view_remove_calendar(WeekView* wv, Calendar* cal);
-int week_view_get_current_week(WeekView* wv);
+struct tm week_view_get_start(WeekView* wv);
+struct tm week_view_get_end(WeekView* wv);
```

Comment:
`struct tm` is more complex than `time_t` and isn't the natural storage format used by `WeekView`. Perhaps these functions should just return the `time_t` and the caller should take care of the conversion if it needs it?